labgrid/driver/power/digipower.py

```diff
@@ -13,14 +13,14 @@ def power_set(host, port, index, value):
 
     suffixstring = "0000000000000000"
     portstring = {
-        "1": "10000000",
-        "2": "01000000",
-        "3": "00100000",
-        "4": "00010000",
-        "5": "00001000",
-        "6": "00000100",
-        "7": "00000010",
-        "8": "00000001",
+        1: "10000000",
+        2: "01000000",
+        3: "00100000",
+        4: "00010000",
+        5: "00001000",
+        6: "00000100",
+        7: "00000010",
+        8: "00000001",
```

Comment:
It's probably better to get rid of this altogether. What about `f'{1 << 8-index:08b}0000000000000000'` instead of `portstring[index] + suffixstring`?

Reply:
Maybe still keep portstring and suffixstring separate for reability. Otherwise the format is much nicer than a dict here.